goty-server/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-all.zip
```

Comment:
Question: 

Idk if -bin vs -all matters, or even what it means.

Reply:
`all` includes the Gradle Build Tool executable, sources, and documentation whereas `bin` is just the executable. `all` is not really needed unless you are doing Gradle development - it just adds to the overall download size. Even then, you can always download the sources separately. However, it also does not hurt to use `all` so don't feel like you need to change it.